paperqa/clients/exceptions.py

```diff
@@ -0,0 +1,4 @@
+class DOINotFoundError(Exception):
```

Comment:
Any chance we can just inline this somewhere? Imo it's not worth its own module

Tho I can also see why it should go in its own module, so feel free to keep here

Reply:
yea I'm gonna keep because I think we're going to add more here